tools/nvqpp/nvq++.in

```diff
@@ -180,10 +180,8 @@ HELP
 }
 
 # Install directory is wherever this script is and up one directory
-install_dir="$(
-	cd -- "$(dirname "$0")/.." >/dev/null 2>&1
-	pwd -P
-)"
+this_file_dir="$(dirname "$(readlink -f "${BASH_SOURCE[0]}")")"
+install_dir="$(dirname "${this_file_dir}")"
```

Comment:
Just curious ... was the prior version of this not working correctly for some cases?